assets/react/search.jsx

```diff
@@ -7,14 +7,8 @@ import 'p2pu-search-cards/dist/build.css';
 
 let elem = document.getElementById('search-lc-component');
 
-let initialState = {}
-if (elem.dataset.teamId) {
-  initialState["team_id"] = elem.dataset.teamId
-}
-
 ReactDOM.render(
   <Search
-    initialState={initialState}
```

Comment:
@s-kennedy this isn't the old search page functionality, this is how the search is scoped by team_id. See [this commit](https://github.com/p2pu/p2pu-website/commit/2cb6de4e407c73c293488890a7c1e0c9cc483009#diff-c752d1a02f7a56eb6f0caf3628086f8b).

Unless you update the search component after that to otherwise get the team ID, I think this will break the search on team pages.

Reply:
ha this is a true face palm moment.